BUG: Relative path for folders wrong

If reporting a bug:

1. __Can you describe the problem and bug in more detail?__
Inside the folder `src/dev/stories/index.js`, these 2 lines of code:
`import ResultItem from '../search-injection/components/ResultItem'`
`import TooltipContainer from '../content-tooltip/components/container'`
wrongly points to the paths of `search-injection` and `content-tooltip` folders.

2. __How can we replicate the issue?__
On running `npm run storybook`, it results into an error while compiling.
`ERROR in ./src/dev/stories/index.js
Module not found: Error: Can't resolve '../content-tooltip/components/container' in '<my_repo_loc>/Memex/src/dev/stories'`

3. __Expected behavior (i.e. solution)__
The file paths should be:
`import ResultItem from '../../search-injection/components/ResultItem'`
`import TooltipContainer from '../../content-tooltip/components/container'`

4. __Error stack (from extension crash page)__
```
ERROR in ./src/dev/stories/index.js
Module not found: Error: Can't resolve '../content-tooltip/components/container' in '/home/kryptoknight/Music/open_source/Memex/src/dev/stories'
 @ ./src/dev/stories/index.js 33:17-67
 @ ./.storybook/config.js
 @ multi ./node_modules/@storybook/core/dist/server/config/polyfills.js ./node_modules/@storybook/core/dist/server/config/globals.js ./.storybook/config.js (webpack)-hot-middleware/client.js?reload=true

ERROR in ./src/dev/stories/index.js
Module not found: Error: Can't resolve '../search-injection/components/ResultItem' in '/home/kryptoknight/Music/open_source/Memex/src/dev/stories'
 @ ./src/dev/stories/index.js 29:18-70
 @ ./.storybook/config.js
 @ multi ./node_modules/@storybook/core/dist/server/config/polyfills.js ./node_modules/@storybook/core/dist/server/config/globals.js ./.storybook/config.js (webpack)-hot-middleware/client.js?reload=true
```

5. __Other comments__

--- a/src/dev/stories/index.js
+++ b/src/dev/stories/index.js
@@ -2,8 +2,8 @@ import React from 'react'
 import { storiesOf } from '@storybook/react'
 import 'babel-polyfill'
 
-import ResultItem from '../search-injection/components/ResultItem'
-import TooltipContainer from '../content-tooltip/components/container'
+import ResultItem from '../../search-injection/components/ResultItem'
+import TooltipContainer from '../../content-tooltip/components/container'
 
 storiesOf('Result Item', module).add('with all props', () => (
     <ResultItem

